### DISCUSSION TITLE
[BP#115] Limiter l'utilisation du Flash

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#15] [Bonnes pratiques] [answered]
La présence de flash sur une page en 2021 est une curiosité.

On peut préciser que flash est indésirable dans « Limiter le recours aux plugins »

**Comment by DocRoms** [comment]
Je suis totalement d'accord : Flash est déprécié et son support à même pris fin le 31 Décembre 2021. 
Il n'est plus supporté par de nombreux navigateurs et mieux vaut ne plus en faire. 

Néanmoins, il arrive encore que certains `intranets` obligent à utiliser une ancienne version de navigateur pour continuer d'accéder à du contenu "Flash"... C'est donc une bonne idée de supprimer cette règle en gardant une mention de l'éviter dans la règle 12  "Limiter le recours au plugins".

**Reply by ACTLEM** [reply]
Effectivement, certains éditeurs n'ont pas encore totalement migrer l'interface en Flash de produits à usage interne sur du HTML5, CSS, JS. J'en ai un gros en tête d'ailleurs.

**Reply by arnolem** [reply]
En complément, je dirais qu'un livre qui parle de "conception" n'a pas pour but de passer en revue les veilles technologies qui pourraient encore être utilisées dans quelques cas marginaux. Aujourd'hui, il est tout simplement impossible de concevoir un projet avec Flash (tout comme Silverlight ou Java embarqué, qui sont cités dans la règle 12).

**Reply by mchoraine** [reply]
J'ai l'impression que cette règle pourrait être supprimée de la même manière que la règle 12. Je me dis que lorsqu'on en vient à s'intéresser aux recommandations d'écoconception, on doit être au courant que ces technologies et donc Flash sont dépréciées et donc, de fait, son utilisation est une mauvaise pratique au delà de l'éco-conception.

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/124
